tests/s3/copy_object/test_copy_obj_simultaneous_del.py

```diff
@@ -91,11 +92,15 @@ def teardown_class(cls):
         LOGGER.info("ENDED: teardown test suite operations.")
 
     @staticmethod
-    def copy_object_wrapper(src_bucket, src_obj, dest_bucket, dest_obj, exception=None):
+    def copy_object_wrapper(src_bucket, src_obj, dest_bucket, dest_obj, exception=None,
+                            expect_copy_fail_always=False):
         """Copy object wrapper for multiprocessing"""
         s3_obj = s3_test_lib.S3TestLib()
         try:
-            return s3_obj.copy_object(src_bucket, src_obj, dest_bucket, dest_obj)
+            resp = s3_obj.copy_object(src_bucket, src_obj, dest_bucket, dest_obj)
+            if expect_copy_fail_always:
```

Comment:
Maybe we can rename this to something on the lines of expect_copy_failure, assert_copy_failure, etc. Having 'always' in the flag name might lead to confusion.

Reply:
renamed to expect_copy_failure .